tests/h2-fuzz/Cargo.toml

```diff
@@ -1,15 +1,15 @@
-[package]
-name = "h2-fuzz"
-version = "0.0.0"
-publish = false
-license = "MIT"
-edition = "2018"
-
-[dependencies]
-h2 = { path = "../.." }
-
-env_logger = { version = "0.9", default-features = false }
-futures = { version = "0.3", default-features = false, features = ["std"] }
-honggfuzz = "0.5"
-http = "0.2"
-tokio = { version = "1", features = [ "full" ] }
+[package]
```

Comment:
Why is the whole file changed? Does this have different (Windows?) line endings, perhaps?

Reply:
Yes, it had DOS EOLs